src/feeds/FeedReader.ts

```diff
@@ -246,7 +246,9 @@ export class FeedReader {
                 const newGuids = [];
                 log.debug(`Found ${feed.items.length} entries in ${url}`);
                 for (const item of feed.items) {
-                    const guid = item.guid || item.id || item.link || item.title;
+                    // Find the first guid-like that looks like a string.
```

Comment:
Being a string doesn't make a value "guid-like".

Reply:
I would hash these before putting in memory. Someone could be a jerk and fill it with MBs of string, and then we'd put MBs into memory :)